src/Aquifer.Public.API/Endpoints/Resources/Collections/List/Request.cs

```diff
@@ -0,0 +1,27 @@
+using Aquifer.Data.Entities;
+using System.ComponentModel;
+
+namespace Aquifer.Public.API.Endpoints.Resources.Collections.List;
+
+public sealed class Request
+{
+    public const int MaximumLimit = 100;
+
+    /// <summary>
+    /// The optional type of resource to search for, such as "Dictionary". If none specified, will default to None.
+    /// </summary>
+    [DefaultValue(ResourceType.None)]
+    public ResourceType ResourceType { get; init; }
+
+    /// <summary>
+    /// The number of results to return.
+    /// </summary>
+    [DefaultValue(MaximumLimit)]
+    public int Limit { get; init; } = MaximumLimit;
```

Comment:
I decided to go ahead and add offset and limit.  Their default values will allow fetching all values but it's future proofing in case we end up with many more parent resources one day.